frontend/src/components/MapView/MapView.tsx

```diff
@@ -36,8 +36,10 @@ const representativePointCircleStyle: MapboxGL.CirclePaint = {
   'circle-radius': {
     property: 'population',
     type: 'exponential',
+    // minPopulation, maxPopulation = 10, 10000
+    // minRadius, maxRadius = 4, 12
```

Comment:
Where is the number 12 from? I do not see it below.

Reply:
Done